LifeCycleAssessment_Engine/Compute/ComputeEquivalency.cs

```diff
@@ -0,0 +1,164 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Base;
+using BH.oM.Reflection.Attributes;
+using BH.oM.Structure.MaterialFragments;
+using BH.oM.LifeCycleAssessment;
+using BH.oM.LifeCycleAssessment.Results;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Returns EPA equivalency using a U.S.A. specific dataset")]
+        [Input("kgCO2", "LCA result for Global Warming Potential")]
+        [Input("equivalencyType", "Resultant unit for equivalency conversion")]
+        [Output("quantityResult", "The numerical value for the resultant unit")]
+        public static double ComputeEquivalency(double kgCO2, EquivalencyType equivalencyType)
```

Comment:
Can we go with a method name that avoids redundancy? EmbodiedCarbonEquivalency perhaps? 
This is bc it already exists in the Compute structure, therefore calling it ComputeSomething is a bit redundant.